andela/ah-scorpion-frontend
# #160311539 Consolidation of Authentication

#### What does this PR do?
Consolidates the authentication stories.

#### Description of Task to be completed?
 - Adds the [homepage](https://github.com/andela/ah-scorpion-frontend/pull/9)
 - Adds the [Sign Up page](https://github.com/andela/ah-scorpion-frontend/pull/16) and allows for user registration
 - Adds the [Login page](https://github.com/andela/ah-scorpion-frontend/pull/14) and allows for login with email and password.
 - Adds the [Social Login buttons](https://github.com/andela/ah-scorpion-frontend/pull/13)
 - Adds the [Password Reset page](https://github.com/andela/ah-scorpion-frontend/pull/15)

#### How should this be manually tested?
 - Please click the links to the above pull request to see how they could be manually tested.

#### What are the relevant pivotal tracker stories?
[#160311539](https://www.pivotaltracker.com/story/show/160311539)
[#160179368](https://www.pivotaltracker.com/story/show/160179368)
[#159993335](https://www.pivotaltracker.com/story/show/159993335)
[#159993337](https://www.pivotaltracker.com/story/show/159993337)
[#159993338](https://www.pivotaltracker.com/story/show/159993338)
[#159993336](https://www.pivotaltracker.com/story/show/159993336)
